index.html

```diff
@@ -9876,8 +9897,11 @@ <h4 id="Beast Shape - Medium Animal">Beast Shape - Medium Animal</h4>
                </td>
                <td>Go for the Throat
                </td>
-               <td>Do a regular melee attack against a target that is either <a class="condition" data-tooltip-container>stunned</a>, <a class="condition" data-tooltip-container>prone</a>,
-                  or Imobolised. On a hit do Heavy+Dex damage
+               <td>Do a Medium+Dex melee attack against a creature.  
+                  If the creature is <a class="condition" data-tooltip-container>stunned</a>, <a class="condition" data-tooltip-container>prone</a>,
+                  <a class="condition" data-tooltip-container>immobilised</a>, <a class="condition" data-tooltip-container>unconscious</a>, or 
+                  <a class="condition" data-tooltip-container>paralysed</a>
+                  this deals Heavy+Dex damage.
```

Comment:
This is more interesting, but maybe needs a little extra situationalness to it? Seems a bit like psionic base ability but easier (aside from being melee).